### PR TITLE
[Image behavior] fixed background jitter fix on mobile

### PR DESCRIPTION
**Pull request by ludoboludo**
### PR Summary: 

On mobile there is a jitter happening on the image due to the address bar showing/hiding depending on your scrolling. 

### Why are these changes introduced?

Fixes https://github.com/Shopify/dawn/issues/2609

### What approach did you take?

Applied a `height: 100vh;` as it doesn't take into account the address bar space being taken. This way there are no jitter when it's appearing and disappearing 👍 

### Testing steps/scenarios

- [ ] Add an image to the image banner section and set `image behavior` to `fixed background position`. 
- [ ] Test it on mobile, ideally both android and iPhone. 

### Demo links

- [Editor](https://os2-demo.myshopify.com/admin/themes/139894456342/editor)

### Checklist
- [ ] Added PR summary for [release notes](https://themes.shopify.com/themes/dawn/styles/default#ReleaseNotes)
- [ ] Requested review from UX (Only for changes that are affecting the experience or perceivable visual details)
- [ ] Created a ticket for the [help.shopify.com](https://help.shopify.com) documentation team about updates to theme settings. (Internal-only task)
- [ ] Followed [theme code principles](https://github.com/Shopify/dawn/blob/main/.github/CONTRIBUTING.md#theme-code-principles)
- [ ] Linted with [Theme Check](https://github.com/Shopify/theme-check)
- [ ] Tested on [mobile](https://shopify.dev/themes/store/requirements#mobile-browser-requirements)
- [ ] Tested on [multiple browsers](https://shopify.dev/themes/store/requirements#desktop-browser-requirements)
- [ ] Tested for [accessibility](https://shopify.dev/themes/best-practices/accessibility)

### CODE REVIEW

**Review by eugenekasimov — APPROVED** [review]

**Review by melissaperreault — APPROVED** [review]
LGTM! 🎉 Tested on both phones works well! Tested with different image ratio too! Good find!

Dumb question: The only thing I wonder is if there will be some incompatibility if we ever offer a new banner height size like a `full height` with the help of `vh` values.

File: assets/base.css [hunk 1]
```diff
@@ -3127,6 +3127,7 @@ details-disclosure > details {
 .animate--fixed > img:not(.zoom):not(.deferred-media__poster-button),
 .animate--fixed > svg:not(.zoom):not(.deferred-media__poster-button) {
   position: fixed;
+  height: 100vh;
```
Comment (kjellr) [inline assets/base.css:3130]: Should we pair this with a `width: auto` to safeguard against stretching if future styles interfere here? I'm always overly cautious about images when it comes to that. 😅

Comment (ludoboludo) [reply assets/base.css:3130]: yeah I agree, I was thinking of that but also not seeing an impact on the images I was testing. Probably safer to add it 👍

Comment (ludoboludo) [reply assets/base.css:3130]: Actually messes with things 😅 and we already have declarations that set the width and max-width to `100%` while the height is set to `100% too. 

https://user-images.githubusercontent.com/28613840/236268356-8d5f4922-ee8d-4b9e-93ad-6c058cd3a51f.mp4

I could also set the height on the image wrapper instead 🤔

Comment (kjellr) [reply assets/base.css:3130]: Oh well. We've got another `width: 100%` rule on the wrapper too, so I'm not sure that's much better. I think what we have here seems to work ok.

**Comment by ludoboludo** [comment]
> LGTM! 🎉 Tested on both phones works well! Tested with different image ratio too! Good find!
> 
> Dumb question: The only thing I wonder is if there will be some incompatibility if we ever offer a new banner height size like a `full height` with the help of `vh` values.

I don't think so as this is only applied when we have a fixed background. And when we do, the image will always be the full size of the viewport (100%). Though that 100% means something different on mobile depending on the browser. So by using 100vh we're making sure the look is consistent on mobile despite the different behaviours of dynamic address bars and what not.

**Review by sofiamatulis — APPROVED** [review]

**Pull request merged**